etl-pipeline/processes/record_clusterer.py

```diff
@@ -30,8 +34,10 @@ class RecordClusterer:
     CLUSTER_SIZE_LIMIT = 10000
     # Regular expression to identify identifiers that should be matched
     IDENTIFIERS_TO_MATCH = r"\|(?:isbn|issn|oclc|lccn|owi)$"
+    CLUSTER_TIMEOUT = 60 * 60 # 1 hour
```

Comment:
Just checking - this is the timeout to cluster a single record?? Seems like if we are reaching anything near that then we probably selected way too many records in the candidate pool identification step

Reply:
ya i think what we were seeing was a record with 2k+ records that represent the work